docs/notebooks/MCMC Sampling.ipynb

```diff
@@ -144,15 +148,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "drawset_pd = sim_data.get_drawset()\n",
-    "drawset_pd.columns\n",
-    "\n",
-    "# restrict to columns over new outcomes of N Bernoulli trials\n",
-    "y_sims = drawset_pd.drop(columns=['lp__', 'accept_stat__'])\n",
+    "y_sims = sim_data.stan_variable('y_sim')\n",
+    "N = y_sims.shape[1]\n",
     "\n",
     "# plot total number of successes per draw\n",
     "y_sums = y_sims.sum(axis=1)\n",
-    "y_sums.astype('int32').plot.hist(range(0,datagen_data['N']+1))"
+    "\n",
+    "import pandas as pd\n",
+    "y_sums_pd = pd.DataFrame(data=y_sums)\n",
+    "y_sums.plot.hist(range(0,N+1))"
```

Comment:
This is fine, I usually use `matplotlib.pyplot as plt` --> `plt.hist(y_syms, bins=range(0,N+1))`

I think using the norm. But nothing needs to be changed here.